src/zzz_od/gui/dialog/app_setting_dialog.py

```
@@ -0,0 +1,64 @@
+        self._layout_inited: bool = False  # 布局是否已经完成初始化
+
+    def show_by_group(self, group_id: str, parent: QWidget) -> None:
+        self.group_id = group_id
+        self.setParent(parent, Qt.WindowType.Dialog)
+        self.init_layout()
+        self.show()
+        self.on_dialog_shown()
+
+    def init_layout(self) -> None:
+        """
+        初始化弹窗的布局，默认是垂直方向的滚动布局
```

Ruff (0.14.7)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)